build/accelerate_launch.py

```diff
@@ -20,23 +20,79 @@
 # Standard
 import os
 import logging
+import subprocess
+import sys
+import traceback
 
 # Third Party
 from accelerate.commands.launch import launch_command
 
 # Local
-from build.utils import process_accelerate_launch_args, get_job_config
+from build.utils import (
+    process_accelerate_launch_args,
+    get_job_config,
+    write_termination_log,
+    USER_ERROR_EXIT_CODE,
+    INTERNAL_ERROR_EXIT_CODE,
+)
 
 
 def main():
     LOGLEVEL = os.environ.get("LOG_LEVEL", "WARNING").upper()
     logging.basicConfig(level=LOGLEVEL)
 
-    job_config = get_job_config()
+    ##########
+    #
+    # Parse arguments
+    #
+    ##########
+    try:
+        job_config = get_job_config()
 
-    args = process_accelerate_launch_args(job_config)
-    logging.debug("accelerate launch parsed args: %s", args)
-    launch_command(args)
+        args = process_accelerate_launch_args(job_config)
+        logging.debug("accelerate launch parsed args: %s", args)
+    except FileNotFoundError as e:
+        logging.error(traceback.format_exc())
+        write_termination_log("Unable to load file: {}".format(e))
+        sys.exit(USER_ERROR_EXIT_CODE)
+    except (TypeError, ValueError, EnvironmentError) as e:
+        logging.error(traceback.format_exc())
+        write_termination_log(
+            "Exception raised during training. This may be a problem with your input: {}".format(
+                e
+            )
+        )
+        sys.exit(USER_ERROR_EXIT_CODE)
+    except Exception as e:  # pylint: disable=broad-except
+        logging.error(traceback.format_exc())
+        write_termination_log("Unhandled exception during training")
```

Comment:
Should this termination log also include e? I see Thara has a similar comment, and I notice a few places where this happens. Do we want to always write `e` to the termination log or is that too long?